src/Navigation/Navigation.tsx

```diff
@@ -8,7 +8,7 @@ import { motion } from "framer-motion"
 import { NavProps } from '../interface'
 
 const Navigation: React.FC<NavProps> = props => {
-  const changeIcon = props.isDarkModeOn? <img src={sunIcon} alt="Toggle Dark and Light Mode" className={"icon" + props.darkMode}/> : 
+  const changeIcon = props.darkMode === '-dark'? <img src={sunIcon} alt="Toggle Dark and Light Mode" className={"icon" + props.darkMode}/> : 
```

Comment:
Thank you for updating this ternary so that the icon changes based on what's being stored in local storage!